### DISCUSSION TITLE
Examen

### DISCUSSION THREAD

**Discussion by andreslb9** [mmasias/23-24-prg2#110] [General]
Hola, los que no estábamos el viernes cuando entregaste los exámenes, dónde vamos el lunes a recogerlo?

**Comment by mmasias** [comment]
Hola Andres

Entregaré los exámenes el lunes a las 14:00h en el aula 1.2

**Comment by andreslb9** [comment]
Mañana salimos a las 12, podría ser antes por favor? Porque hay gente que vive fuera de Santander o que esperar 2 horas les viene mal.
Un saludo

**Comment by mmasias** [comment]
En realidad pensaba entregarlos antes: el viernes en clase de lógica. Y a los pocos que no estuvieran se los enviaba por email. Pero como a la clase de lógica no asistió casi nadie (conté, a ojo, unos 15 alumnos), resultó inviable poder enviarlos por email. 

Nos vemos mañana a las 10:00h en el aula 0.6

**Comment by mmasias** [comment]
Han quedado enviados los cuatro exámenes que restaban recoger.
